demos/embeddings/stellargraph-keras-node2vec.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>This seems like a very few number of epochs - is this usually the case for Node2Vec? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
With enough random walks, Node2Vec will perform good enough. Accurally, in Word2Vec C implementation, only one epoch is required.

Reply:
Ah yep that makes sense, since you can just run more walks to have a larger epoch 👍